tests/test_questionnaire_bank.py

```diff
@@ -551,6 +551,18 @@ def test_visit_6mo(self):
         qbd_i2 = qbd._replace(iteration=1)
         self.assertEqual("Month 18", visit_name(qbd_i2))
 
+    def test_visit_9mo(self):
+        crv = self.setup_qbs()
+        backdate, nowish = associative_backdate(
+            now=now, backdate=relativedelta(months=9))
```

Comment:
Don't like the non-determinism, but out of scope for this PR

Reply:
why not?  this is testing our system far better by moving through the days as the tests run on a nearly constant basis.  if we hard coded dates, it'd only test that one set.

Reply:
It's true we do get more effective coverage, but we might be able to do that with a parameterized test run across a large number of days

Some [strong words from Martin Fowler](https://martinfowler.com/articles/nonDeterminism.html), still need to finish reading...